RecoTracker/PixelSeeding/interface/CACoupleSoA.h

```diff
@@ -0,0 +1,20 @@
+#ifndef RecoTracker_PixelSeeding_interface_CACoupleSoA_h
+#define RecoTracker_PixelSeeding_interface_CACoupleSoA_h
+
+#include <Eigen/Core>
+
+#include <alpaka/alpaka.hpp>
+
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+
+namespace caStructures {
+
+  GENERATE_SOA_LAYOUT(CACoupleLayout, SOA_COLUMN(uint32_t, inner), SOA_COLUMN(uint32_t, outer))
```

Comment:
is a "couple" different from a "pair" ?

Reply:
Not really ... you mean I could use a column of `std::pair`? Or we have something already somewhere?

Reply:
or is it a doublet? a CADoublet?

Reply:
my comment was just about the name: I would prefer to use "doublet" or "pair" consistently, not add another term like "couple"

Reply:
I see the point. Then I would go for `CAPair` to avoid using doublet that for us is specifically a pair of hits.

Reply:
No